test/common/page_objects/add-job-page.js

```diff
@@ -14,6 +14,7 @@ const AddJobPage = function AddJobPage(browser) {
   this.getValidationError = () => browser.text('#validation-errors');
 
   this.visit = (accountId) => browser.visit(`/${accountId}/jobs/new`);
+  this.patch = (accountId, jobId) => browser.visit(`/${accountId}/jobs/${jobId}?_method=PATCH`);
```

Comment:
Yes, i thought you'd notice that ... it fell in line with the "more tests required" line.